README.md

```diff
@@ -1 +1,68 @@
+
+### Benefits
+
+Utilizing Data Feed Proxy Combinators offers several advantages:
+
+- **Enhanced Flexibility:** Empowers dApp developers to tailor data consumption to their precise needs without altering their core logic significantly.
+- **Increased Efficiency:** Reduces the need for redundant on-chain calculations within individual dApps and avoids the operational overhead of requesting numerous slightly varied dAPIs from API providers.
+- **Improved Scalability:** Allows the API3 ecosystem to serve a broader range of data requirements by leveraging existing data feeds in new ways.
+- **Standardization:** Provides a common set of tools for common data feed manipulations, promoting code reuse and reliability.
+- **Simplified Integration:** By adhering to standard interfaces, these combinators lower the barrier for dApps to access derived or transformed data.
+
+## Intended Use Cases
+
+Data Feed Proxy Combinators are particularly useful in scenarios such as:
+
+- **Accessing Inverse Feeds:** When a dApp requires a price quoted as `QUOTE/BASE` but the available oracle feed provides `BASE/QUOTE` (e.g., using `InverseApi3ReaderProxyV1`).
+- **Decimal Precision Adjustment:** Matching a data feed's precision to a dApp's internal numerical representation or the requirements of another protocol (e.g., using `ScaledApi3FeedProxyV1` to convert an 8-decimal feed to an 18-decimal representation).
+- **Calculating Derived Values:**
+  - Cross-Rates: Deriving exchange rates like EUR/JPY from EUR/USD and USD/JPY feeds. This might involve using `InverseApi3ReaderProxyV1` on one feed and then `ProductApi3ReaderProxyV1` with the result and the other feed.
+  - Other Products: Calculating values that are a product of two reported data points.
+- **Adapting External Data Sources:** Integrating data from non-standard on-chain sources (e.g., other oracle networks via `NormalizedApi3ReaderProxyV1`) into systems expecting an API3-compatible feed interface. This is particularly relevant when migrating from other oracle providers or when a specific feed is only available through a non-API3 interface.
+- **Building Complex Financial Instruments:** Creating structured products or derivatives whose values depend on mathematical combinations or transformations of underlying asset prices.
+- **Gas Optimization for Multiple Readers:** If multiple contracts need the same transformed version of a feed, deploying a single combinator proxy can be more gas-efficient than each contract performing the transformation independently.
+
+## Design Philosophy
```

Comment:
Who is this documentation for? The user doesn't care about this too much. If it's for auditing, I think we can move it to a separate md file.